pandas/core/arrays/datetimelike.py

```diff
@@ -553,6 +553,23 @@ def shift(self, periods, freq=None):
         --------
         Index.shift : Shift values of Index.
         """
+        return self._tshift(periods=periods, freq=freq)
+
+    def _tshift(self, periods, freq=None):
+        """
+        Shift each value by `periods`.
+
+        Note this is different from ExtensionArray.shift, which
```

Comment:
how is this relevant here? this is simply a re-definition of what shift means. I understand the desire to make things clear, but this has *always* been the case, for example in comparing a DTI to an Index.